static/styles/footer.css

```diff
@@ -12,3 +12,11 @@ body {
     height: 60px;
     line-height: 60px;
 }
+
+/*
+  Enforces hiding the lastpass autocomplete icon
+  which no longer respects data-lpignore, autocomplete="off"
+*/
+[data-lpignore] + [data-lastpass-icon-root], [data-lpignore="true"] + [data-lastpass-icon-root] {
```

Comment:
This is not really a footer style -- would be more along the lines of "global" or maybe "custom", but I noticed `body` and `html` selectors in here and didn't want to necessarily add a whole extra css file for this work.  Happy to split this out if that's helpful though.